src/uhs/client/client.cpp

```diff
@@ -20,6 +22,43 @@
 #include <utility>
 
 namespace cbdc {
+
+    namespace address {
+        auto decode(const std::string& addr_str)
+            -> std::optional<cbdc::hash_t> {
+            (void)addr_str;
```

Comment:
This snuck past me in #176, but this `void`-cast is non-sensical. The primary reason you'd ever `void`-cast something is to avoid a compiler-complaint for a variable not being used. In C++, it's far preferred to use `[[maybe_unused]]``, but in this case, that's not even relevant (this variable *is* used just a few lines beneath this cast). At least, this should be removed.

Reply:
Good catch, I'm not sure why that was included at all -- removed in latest push